spec/cucumber/glue/proto_world_spec.rb

```diff
@@ -87,16 +92,15 @@ module Glue
 
           define_steps do
             When('{word} {word} {word}') do |subject, verb, complement|
-              log "subject: #{subject}", "verb: #{verb}", "complement: #{complement}", subject: subject, verb: verb, complement: complement
+              log "subject: #{subject}", "verb: #{verb}", "complement: #{complement}"
```

Comment:
Can we change this to be hard-coded. Aruba steps are designed to showcase simple examples and this has 3 parameter types for "I have no idea what reason...." So if this just becomes `When('monkey eats banana')` then we can make the logger step also hard coded and the test showcasing that the logger having 3 items gets translated will be a bit easier to understand

Notwithstanding that, I'm not sure I agree that "is" what happens though. So I'll probably need to dig into this.

Reply:
Updated in 4740677ec62def308572ae349c1c649991222a04.